manifests/profile/pdf_stack.pp

```diff
@@ -34,13 +41,6 @@
     require  => Package[$fonts],
```

Comment:
I see the parse order fix, but this dependency probably doesn't even need to exist.